<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="fileInputTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fileInputButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse..</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="optionsGroupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>611</width>
      <height>701</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QGroupBox" name="colorThresholdingGroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>581</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Color thresholding</string>
     </property>
     <widget class="QSlider" name="colorThresholdSlider">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="colorThresholdSpin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="frameRateGroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>591</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Frame Rate</string>
     </property>
     <widget class="QSlider" name="frameRateSlider">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="frameRateSpin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="comPortsGroupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>591</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>USB Connection</string>
    </property>
    <widget class="QPushButton" name="comPortsListButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>List COM Ports</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comPortsListBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="comPortsConnectButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string> ▶ </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> ▌▌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastForwardButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> ▶▶ </string>
    </property>
   </widget>
   <widget class="QLabel" name="processedVideoLabel">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>429</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="processedVideoLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>591</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
